_data/merchants.yml

```diff
@@ -228,3 +234,96 @@
       url: https://safedice.com
     - name: XMRLotto.com
       url: https://xmrlotto.com
+- category: Artist Stores
```

Comment:
I don't like the idea of adding every single artist. This page is basically text only, adding a list of names will only make  it harder to read and ugly. Can we just put http://www.projectcoralreef.com/all-stores/ , who contains all the links to the artist's shops?

Reply:
Agreed. I'll revert this clutter and add that as a new merchant in the merchant category